internal/directives/schemas/delete-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "DeleteConfig",
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["Path"],
+  "properties": {
+    "Path": {
```

Comment:
This should be lowercase.

Reply:
You marked this as resolved, but did not fix the issue.

Reply:
This is resolved now.